### DISCUSSION TITLE
MFXTooltip - problem with using it

### DISCUSSION THREAD

**Discussion by elfoxus** [palexdev/MaterialFX#347] [Q&A] [answered]
Hi,

I have hard time making MFXTooltip appear. Could anyone provide some code example to help me? Thanks in advance. 👍

**Comment by palexdev** [comment]
Which library? MaterialFX(main branch) or MFXComponents(rewrite branch)
Also, post an MRE if you can, so I can check what you are doing wrong

**Reply by elfoxus** [reply]
I am using 11.16.1 version of you lib.

I can provide some code below, let's assume this in in constructor of another controll I've got in app:

```
MFXTooltip tooltip = new MFXTooltip(this, "tooltip text");
tooltip.install();
```

I can't see no popups appearing on hover etc. What am I missing?

**Reply by palexdev** [reply, marked as answer]
https://github.com/palexdev/MaterialFX/assets/16880178/48aae1a4-0888-49d0-bb52-3f56b1efa608

It works for me. Unless you provide a more complete MRE, I can't help you

**Reply by elfoxus** [reply]
Okay, so I did everything okay. Gotta debug my code. Thanks for example

**Comment by elfoxus** [comment]
What I had problem with was styling. I had to call 'mfxTooltip.setPopupStyleableParent(...)` to make it work. Otherwise no popup was visible.
